pkg/api/v1/schema.go

```diff
@@ -751,6 +751,9 @@ type Interface struct {
 	// If specified, the virtual network interface will be placed on the guests pci address with the specifed PCI address. For example: 0000:81:01.10
 	// +optional
 	PciAddress string `json:"pciAddress,omitempty"`
+	// Backend driver name.
+	// One of: qemu, vhost
+	Driver string `json:"driver,omitempty"`
```

Comment:
should mark that "optional" defaulting to "qumu" for backward compatibility

Reply:
Thanks, will fix it.